src/modules/organization/organization.entity.ts

```diff
@@ -44,6 +44,10 @@ export class Organization implements BaseEnsEntity {
   @Column()
   namespace: string;
 
+  @Field({ nullable: true })
+  @Column({ nullable: true })
+  namehash?: string;
```

Comment:
Similar to application, I think it would be good if the namehash wan't nullable.